src/packages.jl

```diff
@@ -13,9 +13,15 @@ function init_packagemanager()
     # overwrite PKGMAN_DownloadURL
     Globals.MakeReadWriteGlobal(GapObj("PKGMAN_DownloadURL"))
     Globals.PKGMAN_DownloadURL = function(url)
-        # exception handling is omitted by concept, i.e. errors occuring during the download are shown to the user
+      # We catch errors occuring during the download
+      # in order to throw a Julia exception not a GAP error.
+      # If an error occurs then we show it to the user.
+      try
         buffer = Downloads.download(String(url), IOBuffer())
         return GapObj(Dict{Symbol, Any}(:success => true, :result => String(take!(buffer))), recursive=true)
+      catch e
+        rethrow()
+      end
```

Comment:
But that's completely redundant, it does nothing at all. I.e. you could just leave it out.

How about this instead?
```suggestion
      catch e
        GapObj(Dict{Symbol, Any}(:success => false))
      end
```